addons/nuxeo-retention/nuxeo-retention-core/src/main/resources/schemas/retention_definition.xsd

```diff
@@ -31,7 +31,8 @@
       <xs:restriction base="xs:string" ref:resolver="directoryResolver" ref:directory="RetentionEvent" />
     </xs:simpleType>
   </xs:element>
-  <xs:element name="startingPointExpression" type="xs:string" />
+  <xs:element name="startingPointExpressionLanguage" type="xs:string" />
+  <xs:element name="startingPointEventInput" type="xs:string" />
```

Comment:
This seems unnecessarily verbose to me. I would suggest just `startingPointExpression` and `startingPointValue` instead.

Reply:
Yep, it was my first idea and after that I changed it. but let rename them :)